evm/src/main/java/org/hyperledger/besu/evm/precompile/MainnetPrecompiledContracts.java

```diff
@@ -79,11 +79,12 @@ static void populateForByzantium(
     populateForFrontier(registry, gasCalculator);
     registry.put(
         Address.MODEXP,
-        BigIntegerModularExponentiationPrecompiledContract.byzantium(gasCalculator));
-    registry.put(Address.ALTBN128_ADD, AltBN128AddPrecompiledContract.byzantium(gasCalculator));
-    registry.put(Address.ALTBN128_MUL, AltBN128MulPrecompiledContract.byzantium(gasCalculator));
+        new BigIntegerModularExponentiationPrecompiledContract(gasCalculator, Long.MAX_VALUE));
+    registry.put(Address.ALTBN128_ADD, new AltBN128AddPrecompiledContract(gasCalculator, 500L));
+    registry.put(Address.ALTBN128_MUL, new AltBN128MulPrecompiledContract(gasCalculator, 40_000L));
```

Comment:
IMO we should have constants for these values, just for clarity.  arguably the old constructors should have used constants also